src/app/components/ReclaimEscrowForm/index.tsx

```diff
@@ -59,29 +60,24 @@ export const ReclaimEscrowForm = memo((props: Props) => {
   return (
     <Form onSubmit={submit}>
       <Box direction="row" gap="small" pad={{ top: 'small' }}>
-        <Box background="background-front">
+        <Box>
           <Box width="small">
-            <TextInput
-              data-testid="amount"
-              id="amount-id"
-              name="amount"
-              placeholder={t('common.amount')}
-              type="number"
-              step="any"
-              min={0.0001}
+            <AmountTextInput
+              availableAmount={props.maxAmount}
```

Comment:
potential confusion: min, max, value are in ROSE (or similar) units, availableAmount is in base units

actually are there use cases where this is not equal to the max?

Reply:
good point and thanks for help  I will circle back to the issue once oasis scan work is done 🖖

Reply:
- redundant availableAmount prop is removed
- min, max, value are passed to a new component in ROSE units